packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -898,6 +898,8 @@
 	"nodeCreator.subcategoryNames.tools": "Tools",
 	"nodeCreator.subcategoryNames.vectorStores": "Vector Stores",
 	"nodeCreator.subcategoryNames.miscellaneous": "Miscellaneous",
+	"nodeCreator.sectionNames.popular": "Popular",
```

Comment:
should we add subcategory to that path?
`"nodeCreator.sectionNames.tools.popular": "Popular",`

Reply:
Personally I like it without because the section names are reused for multiple subcategories (like popular,other)